standalone/src/models/commons/resource-identifier/fields-config.ts

```diff
@@ -0,0 +1,25 @@
+import { fake, type TModelFieldsConfig } from '@/core';
+import type {
+  TResourceIdentifierGraphql,
+  TResourceIdentifierRest,
+} from './types';
+
+const commonFieldsConfig = {
+  key: fake((f) => f.string.uuid()),
+  id: fake((f) => f.string.uuid()),
```

Comment:
this might be a bit misleading, it is either id or (whenever supported) key.
to model that, it might be possible to drop key on post build whenever id exists.